gittyup.nim

```diff
@@ -911,12 +694,15 @@ proc message*(thing: GitThing): string =
   ## retrieve the message associated with a git thing
   assert thing != nil and thing.o != nil
   case thing.kind:
-  of goTag:
+  of GIT_OBJECT_TAG:
     result = cast[GitTag](thing.o).message
-  of goCommit:
+  of GIT_OBJECT_COMMIT:
     result = cast[GitCommit](thing.o).message
   else:
-    raise newException(ValueError, "dunno how to get a message: " & $thing)
+    raise newException(
+      ValueError,
+      "Cannot get message for git object " &
+        $thing & " (kind was '" & $thing.kind & "')")
```

Comment:
Let's use `raise ValueError.newException:` syntax so we don't need to concern ourselves with the dangling `)`.